projects/wikimedia.org.yaml

```diff
@@ -8,33 +8,44 @@ paths:
             title: Wikimedia REST API
             description: >
                 This API provides cacheable and straightforward access to
-                Wikimedia content and data, in machine-readable formats. Each
-                entry point has explicit stability markers to inform you about
-                development status and change policy, according to [our API
-                version policy](https://www.mediawiki.org/wiki/API_versioning).
+                Wikimedia content and data, in machine-readable formats.
 
-                See https://www.mediawiki.org/wiki/REST_API for background and details.
+                ### Global Rules
 
-                ### High-volume access
-                  - Limit your clients to no more than 200 requests/s to this API.
-                  - Set a unique `User-Agent` or `Api-User-Agent` header that
-                    allows us to contact you quickly. Email addresses or URLs
-                    of contact pages work well.
+                - Limit your clients to no more than 200 requests/s to this API.
+                  Each API endpoint's documentation may detail more specific usage limits.
+                - Set a unique `User-Agent` or `Api-User-Agent` header that
+                  allows us to contact you quickly. Email addresses or URLs
+                  of contact pages work well.
 
                 By using this API, you agree to Wikimedia's 
                 [Terms of Use](https://wikimediafoundation.org/wiki/Terms_of_Use) and
-                [Privacy Policy](https://wikimediafoundation.org/wiki/Privacy_policy), and
-                you irrevocably agree to release modifications or additions
-                made through this API under the 
+                [Privacy Policy](https://wikimediafoundation.org/wiki/Privacy_policy).
+                Unless otherwise specified in the endpoint documentation
+                below, content accessed via this API is licensed under the
                 [CC-BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/) 
-                License and the [GFDL](https://www.gnu.org/copyleft/fdl.html).
+                and [GFDL](https://www.gnu.org/copyleft/fdl.html) licenses,
+                and you irrevocably agree to release modifications or
```

Comment:
I think this needs to be "and you irrevocably agree to release modifications or additions made through this API under the CC-BY-SA 3.0 license and the GFDL."